docs/cookbook/seed_phrase.md

```diff
@@ -0,0 +1,38 @@
+# Starting from a Seed Phrase
+
+In order to generate a wallet from a seed phrase, it is necessary to generate descriptors from the seed phrase first. The extended key can be generated from the seed phrase and used with a descriptor template to build a wallet.
+
+Note - Requires `keys-bip39` feature in `bdk_wallet`
```

Comment:
This can be an admonition, check the docs [here](https://squidfunk.github.io/mkdocs-material/reference/admonitions/)

```suggestion
!!! note "Feature Flags"

    The current example requires the feature `keys-bip39` for `bdk_wallet`.
    You can add with:
    
    ```bash
    cargo add bdk_wallet --features="keys-bip39"
    ```
```

We use them across the cookbook, see https://github.com/bitcoindevkit/book-of-bdk/blob/fcbec7088e92cbe97e7fbdf54c9c3772a0fcd6e6/docs/cookbook/descriptors.md?plain=1#L4-L5 for an example